### ISSUE TITLE
Images with "--" in the name don't register Push and Pull time properly

### ISSUE THREAD

**Issue by patrickleboutillier** [goharbor/harbor#10623]
Here is an example:

```
[root@ref2docker ~]# docker images
REPOSITORY                                                                           TAG                 IMAGE ID            CREATED             SIZE
srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653   latest              1c6ee342054e        36 minutes ago      1.6GB
srvharbor.croesus.local/references/ref90-15-pf-2                                     latest              1c6ee342054e        36 minutes ago      1.6GB

[root@ref2docker ~]# docker push srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653
The push refers to repository [srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653]
41f6e8ed8e0c: Layer already exists
latest: digest: sha256:fb621c31dca183abfce683d7b5bb4587b97135a4bc4c107a5ab5dae2dda9d510 size: 530
[root@ref2docker ~]# docker push srvharbor.croesus.local/references/ref90-15-pf-2
The push refers to repository [srvharbor.croesus.local/references/ref90-15-pf-2]
41f6e8ed8e0c: Layer already exists
latest: digest: sha256:fb621c31dca183abfce683d7b5bb4587b97135a4bc4c107a5ab5dae2dda9d510 size: 530

[root@ref2docker ~]# docker rmi srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653
Untagged: srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653:latest
Untagged: srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653@sha256:fb621c31dca183abfce683d7b5bb4587b97135a4bc4c107a5ab5dae2dda9d510
[root@ref2docker ~]# docker rmi srvharbor.croesus.local/references/ref90-15-pf-2
Untagged: srvharbor.croesus.local/references/ref90-15-pf-2:latest
Untagged: srvharbor.croesus.local/references/ref90-15-pf-2@sha256:fb621c31dca183abfce683d7b5bb4587b97135a4bc4c107a5ab5dae2dda9d510
Deleted: sha256:1c6ee342054e884119995c470eea33d2cdbb4ccfcbbd87a121bb724faf7a2814
Deleted: sha256:1dd978fd973cb51ca2ba613931a0c61ef931c22d80cf149fbc7dae1002ead359
Deleted: sha256:d92eb2c5c098a319d2dbd70410856a45e4f17482e4e5dd1c6ff0e35eead196b5
Deleted: sha256:41f6e8ed8e0c4e4f24ee1787f4629ed0fd774a5cfc63f88a4c6737d734669479

[root@ref2docker ~]# docker pull srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653
Using default tag: latest
latest: Pulling from references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653
c045794a45e1: Pull complete
Digest: sha256:fb621c31dca183abfce683d7b5bb4587b97135a4bc4c107a5ab5dae2dda9d510
Status: Downloaded newer image for srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653:latest
srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653:latest
[root@ref2docker ~]# docker pull srvharbor.croesus.local/references/ref90-15-pf-2
Using default tag: latest
latest: Pulling from references/ref90-15-pf-2
Digest: sha256:fb621c31dca183abfce683d7b5bb4587b97135a4bc4c107a5ab5dae2dda9d510
Status: Downloaded newer image for srvharbor.croesus.local/references/ref90-15-pf-2:latest
srvharbor.croesus.local/references/ref90-15-pf-2:latest

[root@ref2docker ~]# docker images
REPOSITORY                                                                           TAG                 IMAGE ID            CREATED             SIZE
srvharbor.croesus.local/references/ref90-15-pf-2                                     latest              1c6ee342054e        43 minutes ago      1.6GB
srvharbor.croesus.local/references/ref90-15-pf-2--webinv-v9-croesus-co7x-1_8_2_653   latest              1c6ee342054e        43 minutes ago      1.6GB
```

![image](https://user-images.githubusercontent.com/3837212/73694589-5b63bd00-46a6-11ea-9682-c3944b8e6b6f.png)
![image](https://user-images.githubusercontent.com/3837212/73694615-6a4a6f80-46a6-11ea-9199-d1dd3a40eebd.png)
![image](https://user-images.githubusercontent.com/3837212/73694360-e4c6bf80-46a5-11ea-8b75-b5cd29437f03.png)

**Comment by bitsf** [comment]
It seems upload time info not recorded in db

**Comment by wy65701436** [comment]
it's fixed in Harbor v2.0.0, and RC build will be ready soon.
